sql/core/src/main/scala/org/apache/spark/sql/execution/ExistingRDD.scala

```diff
@@ -169,10 +171,12 @@ case class LogicalRDD(
 case class RDDScanExec(
     output: Seq[Attribute],
     rdd: RDD[InternalRow],
-    override val nodeName: String,
+    name: String,
     override val outputPartitioning: Partitioning = UnknownPartitioning(0),
     override val outputOrdering: Seq[SortOrder] = Nil) extends LeafExecNode {
 
+  override val nodeName: String = s"Scan RDD $name ${output.map(_.name).mkString("[", ",", "]")}"
```

Comment:
ditto

Reply:
removed `output`. The `name` in there would help in identifying the nodes uniquely